srml/elections/src/lib.rs

```diff
@@ -375,7 +375,7 @@ decl_module! {
 			#[compact] assumed_vote_index: VoteIndex
 		) {
 			let reporter = ensure_signed(origin)?;
-			let who = T::Lookup::lookup(who)?;
+			let who = T::Lookup::lookup(who).ok_or("Could not find `who` account")?;
```

Comment:
why?

Reply:
By "why" you mean, "why this custom error message?". I changed `Lookup` to return an `Option` instead of a `Result`. Otherwise we would require a `type Error` in the trait and would need to make sure that this is convertible into the module error. Given the fact that a lookup fails in 99% of the cases, because it could not find the item it should lookup, I think we can go with the `Option` as return value.

Reply:
Could there not be a `LookupError` value common to all modules?